.github/workflows/run-integration-tests.yml

```diff
@@ -34,12 +36,13 @@ jobs:
             objects.githubusercontent.com:443
             oss.sonatype.org:443
             raw.githubusercontent.com:443
+            registry-1.docker.io:443
```

Comment:
Added based on [this report](https://app.stepsecurity.io/github/PicnicSupermarket/error-prone-support/actions/runs/12464380205?jobid=34788363557&tab=network-events).